### ISSUE TITLE
[AH-38] Reconsider Reset styles file

### ISSUE THREAD

**Issue by Shagon1k** [Shagon1k/AHurynovich-CV#91]
### Description
Nowadays reset.scss should be overthinked in order to correctly reset common css rules without harm to A11y.
